src/main/kotlin/com/project/goms/domain/studentCouncil/presentation/StudentCouncilController.kt

```diff
@@ -61,4 +62,9 @@ class StudentCouncilController(
             .let { studentCouncilConverter.toResponse(it) }
             .let { ResponseEntity.ok(it) }
 
+    @DeleteMapping("outing-list/{accountIdx}")
```

Comment:
리스트 전체를 삭제하는게 아닌 외출자 한명을 삭제하는건데 outing-list 보다는 outing이 좋을거 같습니다.
```suggestion
    @DeleteMapping("outing/{accountIdx}")
```

Reply:
수정했습니다